docs/features/service_tags.md

```diff
@@ -9,7 +9,9 @@ source used. For Consul, tags are supported out-of-the-box.
 
 For this feature to work correctly, you have to extract a tag from request
 data in some way.
-To achieve it, appropriate static Envoy config is required.
+When using listeners configured via EC (`envoy-control.envoy.snapshot.dynamicListeners.enabled` set to `true`)
+a default working config will be provided (see: `EnvoyListenersFactory.kt` for details).
+If not an appropriate static Envoy config is required.
```

Comment:
why is this repeated in service tags docs?

Reply:
Because both canary and service tag filters were moved from static config files to dynamic configuration via API.